athenahealth/appointments.go

```diff
@@ -624,6 +628,56 @@ func (h *HTTPClient) BookAppointment(ctx context.Context, patientID, apptID stri
 	return out[0], nil
 }
 
+type UpdateBookedAppointmentOptions struct {
+	// New appointment type ID for this appointment.
+	AppointmentTypeID *string `json:"appointmenttypeid"`
+	// New department ID for this appointment.
+	DepartmentID *string `json:"departmentid"`
+	// New provider ID for this appointment.
+	ProviderID *string `json:"providerid"`
+	// New supervisingprovider ID for this appointment.
+	SupervisingProviderID *string `json:"supervisingproviderid"`
+}
+
+// https://docs.athenahealth.com/api/api-ref/appointment-booked#Appointment-Booked
+// status	string	This subroutine will return 1 on success, and will otherwise return an error message.
+var successfulResponse = "1"
```

Comment:
We might want to scope this name to appointments (like `successfulBookedApptResponse`) until we know if this is a broader pattern but not blocking